modules/__tests__/AsyncProps-test.js

```diff
@@ -395,6 +395,33 @@ describe('AsyncProps', () => {
         />
       ), div, next)
     })
+
+    it('allows to override resolver function', (done) => {
+      const appSpy = spyOn(App, 'loadProps').andCallThrough()
+
+      const next = execNext([
+        () => {},
+        () => {
+          expect(appSpy.calls.length).toEqual(1)
+          expect(appSpy.calls[0].arguments[2]).toEqual('check')
+        },
+        done
+      ])
+
+      function resolver(Component, params, cb) {
+        Component.loadProps(params, cb, 'check')
```

Comment:
https://github.com/rackt/async-props/pull/42#discussion-diff-49989357R406

this will work only with resolver feature